doc/sphinx/practical-tools/utilities.rst

```diff
@@ -305,10 +305,10 @@ section of the generated makefile. These include:
    override the flags passed to ``coqdoc``. By default ``-interpolate -utf8``.
 :COQDOCEXTRAFLAGS:
    extend the flags passed to ``coqdoc``
-:COQLIBINSTALL, COQDOCINSTALL:
-   specify where the Coq libraries and documentation will be installed.
+:COQLIBINSTALL, COQPLUGININSTALL, COQDOCINSTALL:
+   specify where the Coq libraries, plugins and documentation will be installed.
    By default a combination of ``$(DESTDIR)`` (if defined) with
-   ``$(COQLIB)/user-contrib`` and ``$(DOCDIR)/user-contrib``.
+   ``$(COQLIB)/user-contrib``, ``$(COQLIB)/..`` and ``$(DOCDIR)/user-contrib``.
```

Comment:
yes but coqcorelib is not part of the doc, and given the .. the result is the same.

Reply:
On a sane operating system, you should expect `$COQLIB` to be `/usr/lib/coq` and `$COQCORELIB/..` to be `/usr/lib/ocaml`. So, the result is not the same.

Reply:
right, if we had OCAMMLIB we could use that in the explanation... concretely what do you suggest @CohenCyril does?

Reply:
I suggest to make it just the same as the code, that is, `COQDOCINSTALL` is `$(DOCDIR)/coq/user-contrib` by default, and `COQPLUGININSTALL` is `$(COQCORELIB)/..`. Talking about `COQCORELIB` does not seem much of an issue, since it is displayed by `make printenv` anyway.